packages/manager/apps/telecom/src/app/telecom/telephony/alias/changeType/telecom-telephony-alias-changeType.html

```diff
@@ -110,6 +110,7 @@
                         class="transclude-image-container"
                     >
                         <img
+                            class="w-100"
```

Comment:
I cannot reproduce the issue, tested on both with Firefox and Chromium.

Furthermore there are several `<img>` tag in this following template. It only affect `schema-conference.svg`?

Is is linked to this rules: https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/app/telecom/telephony/telecom-telephony.less#L46?

Reply:
I thought it was a ng-repeat, I badly read the code, my bad 
I can reproduce on Chromium from `feat/solarsteein`, maybe I need to update my branch